src/simulations/tests/simulator_test.py

```diff
@@ -99,7 +120,7 @@ def test_simulator_run_one_with_estimate_noiser(self):
     sim = get_simple_simulator(sketch_estimator_config)
     data_frame = sim.run_one()
     self.assertLen(data_frame, 1)
-    self.assertEqual(data_frame['estimated_cardinality'].iloc[0], 10)
+    self.assertEqual(data_frame['estimated_cardinality_1'].iloc[0], 10)
```

Comment:
I think it is my problem when I originally wrote the code that uses the hard coded column name ('estimated_cardinality') rather than using the global variable.
Matthew, do you think that it makes sense to use global variable (i.e., ESTIMATED_CARDINALITY_BASENAME) and hereafter in this file? So if we changed the name in the future, it would involve less code change and have better maintenance.

Reply:
Done.